packages/cli/src/commands/generate/sdl/templates/sdl.js.template

```diff
@@ -2,15 +2,13 @@ export const schema = gql`
   type ${singularPascalName} {
     ${query}
   }
-
-  <% enums.forEach((enumDef, idx) => { %>
-    enum ${enums[idx].name} {
-      <% enums[idx].values.forEach((enumDefValue, idk) => { %>
-          ${enums[idx].values[idk].name}
-      <% }) %>
-    }
-  <% }) %>
-
+<% enums.forEach((enumDef, idx)=> { %>
+  enum ${enums[idx].name} {
+<% enums[idx].values.forEach((enumDefValue, idk)=> { %>
+    ${enums[idx].values[idk].name}
+<% }) %>
+  }
+<% }) %>
```

Comment:
Should this be changed to this to remove the extra lines?

```suggestion
<% enums.forEach((enumDef, idx)=> { %>
  enum ${enums[idx].name} {
<% enums[idx].values.forEach((enumDefValue, idk)=> { %>${enums[idx].values[idk].name}<% }) %>
  }
<% }) %>
```

Reply:
The above suggestion unfortunately generates all the definitions on the same line. I have not been able to find anything nicer than the above, as the `forEach` construct introduces newlines.

Reply:
Try moving that closing `<% }) %>` for the second `forEach` to a new line.

I know it sucks, the templates end up being ugly as hell, but we'd much prefer the output to be nice and clean than the templates being nice and clean.